uint32 encoder_position
int16 SteeringSetPoint
int16 SteeringPWM
time time_received
